app/src/main/java/com/onix/internship/ui/notesList/addNotes/AddNoteFragmentModel.kt

```diff
@@ -0,0 +1,12 @@
+package com.onix.internship.ui.notesList.addNotes
+
+import androidx.databinding.ObservableBoolean
+import androidx.databinding.ObservableField
+import com.onix.internship.data.NotesColor
+
+data class AddNoteFragmentModel(
+    val title: ObservableField<String> = ObservableField(""),
```

Comment:
no need in special AddNoteFragmentModel - just one model in database
and one model for all - and a little mapper between